attributes/default.rb

```diff
@@ -47,29 +48,25 @@
 default['php']['fpm_ini_control'] = false
 
 case node['platform_family']
-when 'rhel', 'fedora', 'amazon'
+when 'rhel', 'amazon'
   lib_dir = node['kernel']['machine'] =~ /x86_64/ ? 'lib64' : 'lib'
+
   default['php']['conf_dir']      = '/etc'
   default['php']['ext_conf_dir']  = '/etc/php.d'
   default['php']['fpm_user']      = 'nobody'
   default['php']['fpm_group']     = 'nobody'
   default['php']['fpm_listen_user']   = 'nobody'
   default['php']['fpm_listen_group']  = 'nobody'
   default['php']['ext_dir']           = "/usr/#{lib_dir}/php/modules"
-  if platform?('amazon') # amazon names their packages with versions on 201X amazon
-    default['php']['src_deps'] = %w(bzip2-devel libc-client-devel curl-devel freetype-devel gmp-devel libjpeg-devel krb5-devel libmcrypt-devel libpng-devel openssl-devel t1lib-devel libxml2-devel libxslt-devel zlib-devel)
+  default['php']['fpm_package']       = 'php-fpm'
 
-    if node['platform_version'].to_i == 2
-      default['php']['packages']      = %w(php php-devel php-pear)
-      default['php']['fpm_package']   = 'php-fpm'
-    else
-      default['php']['packages']      = %w(php56 php56-devel php-pear)
-      default['php']['fpm_package']   = 'php56-fpm'
-    end
+  # amazon names their packages with versions on 201X amazon
```

Comment:
if we're nuking CentOS 6 then we should nuke 201x amazon. It's the same thing

Reply:
I did, that comment should have gone away as well, thanks for catching it!